shared/css/professional_learning_marketing_banner.scss

```diff
@@ -21,6 +21,7 @@ $dark_blue_background_colour: #0d4172;
 
 .homepage-professional-learning-banner {
```

Comment:
I just did a quick search for this class and discovered it's also used in `homepage_below_hero_trophy_banner.haml`. I have a slight preference towards hiding this with a boolean check like in https://github.com/code-dot-org/code-dot-org/blob/staging/pegasus/sites.v3/code.org/views/homepage_below_hero_trophy_banner.haml#L1 so that there's no concerns that the changes in this class will affect another banner and also less unnecessary content in our DOM is good. Let me know if there's a specific reason that you decided to hide the banner with css

Reply:
Done and ready for review — thanks for the help! 😄